added glossary entries (#432)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc

*See also*:

[[hashbang]]
==== image:images/no.png[no] hashbang (noun)

> *Description*: Do not use to refer to a computer language construct that controls which interpreter parses and interprets instructions in a computer program. Instead, on first mention include both "interpreter directive" and "shebang", for example, "Interpreter directives, also known as shebangs ...".

Reviewer: ```suggestion
*Description*: Do not use this noun when referring to a computer language construct that controls which interpreter parses and interprets instructions in a computer program. Instead, on first mention include both "interpreter directive" and "shebang". For example, "Interpreter directives, also known as shebangs ...".
```

> *Description*: Do not use this noun when referring to a computer language construct that controls which interpreter parses and interprets instructions in a computer program. Instead, on first mention include both "interpreter directive" and "shebang". For example, "Interpreter directives, also known as shebangs ...".

Reviewer 2: ```suggestion
*Description*: Do not use this noun when referring to a computer language construct that controls which interpreter parses and interprets instructions in a computer program. Instead, on the first mention, include both "interpreter directive" and "shebang." For example, "Interpreter directives, also known as shebangs ...".
```

> *Description*: A _shebang_ such as `#!/bin/bash` is a common term for an _interpreter directive_, which is a computer language construct that controls which interpreter parses and interprets instructions in a computer program. This term is also commonly known as "shebang...". Do not use "hashbang". Instead,  on first mention include both "interpreter directive" and "shebang", for example, "Interpreter directives, also known as shebangs".

Reviewer 3: ```suggestion
*Description*: A _shebang_ such as `#!/bin/bash` is a common term for an _interpreter directive_, which is a computer language construct that controls which interpreter parses and interprets instructions in a computer program. This term is also commonly known as "shebang...". Do not use "hashbang". Instead, on first mention include both "interpreter directive" and "shebang", for example, "Interpreter directives, also known as shebangs".
```
